dist/components/Container.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference types="react" />
```

Comment:
Can you uncommit the generated declaration files? They are meant to be generated only in the output directory.

Reply:
@aravi1626